flink-formats/flink-sql-protobuf/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,9 @@
+flink-sql-protobuf
+Copyright 2014-2021 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
```

Comment:
Besides, 'protobuf-java' does not use 'Apache Software License 2.0', it uses 'BSD-3 License'

Reply:
@libenchao thanks for your comment. I have fixed it.